src/Symfony/Component/Messenger/Transport/Sender/SendersLocatorInterface.php

```diff
@@ -29,7 +29,7 @@
      * @param bool|null &$handle True after calling the method when the next middleware
      *                           should also get the message; false otherwise
      *
-     * @return iterable|SenderInterface[]
+     * @return iterable|SenderInterface[] Indexed by sender alias if available
```

Comment:
isnt there always an index available now?

Reply:
No, it depends on the provided `$senders` in the construct. Aliases are completely optional. Index is not relevant if no alias provided.